src/filterdiff.c

```diff
@@ -155,27 +156,64 @@ static int file_exists (const char *name, const char *timestamp)
 static int output_header_line (const char *line)
 {
 	char *fn;
-	int h = strcspn (line + 4, "\t\n");
 
-	fwrite (line, 1, 4, stdout);
-
-	if (prefix_to_add)
-		fputs (prefix_to_add, stdout);
-	else {
-		if (old_prefix_to_add && strncmp (line, "---", 3) == 0)
-			fputs (old_prefix_to_add, stdout);
-		if (new_prefix_to_add && strncmp (line, "+++", 3) == 0)
-			fputs (new_prefix_to_add, stdout);
-	}
+	if (strncmp(line, "diff", 4) == 0 && isspace(line[4])) {
```

Comment:
The project style is to have a space between the function name and the opening parenthesis.